src/features/loginPage/LoginPage.tsx

```diff
@@ -1,5 +1,167 @@
+/* eslint-disable import/named */
+
+import * as Yup from 'yup';
+import toast from 'react-hot-toast';
+import { ErrorMessage, Field, Form, Formik, FormikHelpers } from 'formik';
+import { Link, useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import { RootState } from '../../app/store';
+import { authActions } from './authSlice';
+import { userActions } from './userSlice';
+
+interface IFormValues {
+  email: string;
+  password: string;
+}
 const LoginPage = () => {
-  return <h1 className="p-8 text-center text-6xl">Login Page</h1>;
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const isAuth = useSelector(
+    (state: RootState) => state.authentication.isAuthenticated,
+  );
+  const { t } = useTranslation('login');
+
+  const loginSchema = Yup.object().shape({
+    email: Yup.string().email(t('emailError')).required(t('emailError2')),
+    password: Yup.string().required(t('password2')),
+  });
+
+  const loginHandler = async (data: object) => {
+    const toastId = toast.loading('Loading...');
+    const options = {
+      method: 'POST',
+      body: JSON.stringify(data),
+      headers: { 'Content-Type': 'application/json' },
+    };
+
+    try {
+      const res = await fetch(
+        'https://s-mallets-backend.vercel.app/auth/login',
+        options,
+      );
+
+      if (res.status !== 200) {
+        toast.error(t('toastBad'), {
+          id: toastId,
+        });
+      }
+
+      const resJson = await res.json();
+
+      if (res.status === 200) {
+        dispatch(authActions.login());
+        dispatch(
+          userActions.user({
+            email: resJson.email,
+            name: resJson.name,
+            lastname: resJson.lastname,
+            roles: resJson.roles,
+            phoneNumber: resJson.phoneNumber,
+          }),
+        );
+
+        toast.success(t('toastOk'), {
+          id: toastId,
+        });
+        navigate('/user');
+      }
+    } catch (error) {
+      toast.error(t('toastBad'));
+    }
+  };
+
+  const logoutHandler = () => {
+    dispatch(authActions.logout());
```

Comment:
You shuold also probably clean user data in the store after logout

Reply:
Probably it's not exactly my scope :P but i'll change a bit userSlice also ;)

Reply:
done ;P